src/components/UniversalNavbarExpanded/DropdownNav/DropdownCta.js

```diff
@@ -16,24 +16,30 @@ import IconIntegratedTitle from './IconIntegratedTitle'
  *
  * @param {string} title - A short title
  * @param {string} subcopy - A sentence of supporting copy
+ * @param {element | boolean} alternateIcon - Alternate icon to display in place of DropdownLinkIcon
  *
  * @return {JSX.Element}
  */
-const DropdownCta = ({ title, subcopy }) => (
-  <>
-    <TitleSmall.Serif.Book500>
-      <IconIntegratedTitle title={title}>
-        <DropdownLinkIcon />
-      </IconIntegratedTitle>
-    </TitleSmall.Serif.Book500>
-    <Spacer.H8 />
-    <Body.Regular400 color={COLORS.GRAY_SECONDARY}>{subcopy}</Body.Regular400>
-  </>
-)
+const DropdownCta = ({ title, subcopy, alternateIcon }) => {
+  const Icon = alternateIcon ? alternateIcon : DropdownLinkIcon
```

Comment:
`alternateIcon` needs to be uppercased to `Icon` to become a valid component element in JSX that meets the prop type requirement of `IntegratedIconTitle`.

Passing in lowercase `alternateIcon` returns `objects are not valid as a React child` error.